### PR TITLE
add pynput

### PR DESCRIPTION
**Pull request by ndmaxar**
This PR is for pynput. There is another pynput PR open here: https://github.com/conda-forge/staged-recipes/pull/10568. It has not been updated in a month so I am opening this. I will mention the opening of this PR in the other.
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ndmaxar** [comment]
@oblute @rluria14 ok to be maintainers?

**Comment by ndmaxar** [comment]
@conda-forge/staged-recipes can you help? The appveyor build continues to fail despite all other builds succeeding?

**Comment by synapticarbors** [comment]
If you look at the error message in the log (https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/32047650#L1212), you'll see that the issue is that `bash` is not a command that is recognized on the Windows build system. 

You'll need an equivalent command using the dos command prompt commands and then stick a  `# [win]` at the end of that test line and a `# [unix]` at the end of the current test.

**Comment by xhochy** [comment]
AppVeyor failing is ok as long as Azure Windows is working.

File: recipes/pynput/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - setuptools-lint >=0.5
+    - sphinx 
+    - wrapt ==1.11.2
+    - pyobjc-framework-quartz  # [osx]
+  run:
+    - python 
+    - six
+    - setuptools-lint >=0.5
+    - sphinx
+    - wrapt ==1.11.2
```
Comment (xhochy) [inline recipes/pynput/meta.yaml:29]: If the pinning needs to be that strict, use the following syntax.
```suggestion
    - wrapt 1.11.2.*
```

File: recipes/pynput/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
+    - setuptools-lint >=0.5
+    - sphinx 
+    - wrapt ==1.11.2
+    - pyobjc-framework-quartz  # [osx]
```
Comment (xhochy) [inline recipes/pynput/meta.yaml:23]: Are there required to build the package or can they be in `run` only?

File: recipes/pynput/meta.yaml (hunk 1 above)
Comment (ndmaxar) [reply recipes/pynput/meta.yaml]: I've updated it.

File: recipes/pynput/meta.yaml (hunk 2 above)
Comment (ndmaxar) [reply recipes/pynput/meta.yaml]: I've checked each and few aren't necessary for `host`.

**Pull request merged**